dotnetv3/S3/non-versioned-examples/DeleteMultipleObjectsExample/DeleteMultipleObjects.cs

```diff
@@ -0,0 +1,130 @@
+﻿// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier:  Apache-2.0
+
+/// <summary>
+/// This example shows how to delete multiple objects from an Amazon Simple
+/// Storage Service (Amazon S3) bucket. The example was created using the
+/// AWS SDK for .NET 3.7 and .NET Core 5.0.
+/// </summary>
+namespace DeleteMultipleObjectsExample
+{
+    // snippet-start:[S3.dotnetv3.DeleteMultipleObjectsExample]
+    using System;
+    using System.Collections.Generic;
+    using System.Threading.Tasks;
+    using Amazon.S3;
+    using Amazon.S3.Model;
+
+    public class DeleteMultipleObjects
+    {
+        /// <summary>
+        /// The Main method initializes the Amazon S3 client and the name of
+        /// the bucket and then passes those values to MultiObjectDeleteAsync.
+        /// </summary>
+        public static async Task Main()
+        {
+            const string bucketName = "doc-example-bucket";
+
+            // If the Amazon S3 bucket from which you wish to delete objects is not
+            // located in the same AWS Region as the default user, specify the
+            // bucket Region as a parameter to the client constructor.
```

Comment:
specify the bucket Region -> define the AWS Region for the Amazon S3 bucket

Reply:
changed